doc/reference/tooling/tt_cli/export.rst

```diff
@@ -0,0 +1,145 @@
+.. _tt-export:
+
+Exporting data
+==============
+
+..  admonition:: Enterprise Edition
+    :class: fact
+
+    This command is supported by the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+
+
+..  code-block:: console
+
+    $ tt [crud] export URI FILE SPACE [EXPORT_OPTION ...]
+
+``tt [crud] export`` exports a space's data to a file.
```

Comment:
Not sure if possessive is correct for this case (`space's data`).
Maybe rephrase? 

```suggestion
``tt [crud] export`` exports data from a given space to a file.
```

Reply:
Also, maybe mention the file format in this general description?

Reply:
> a space's data

I believe, possessive works good for our `space` term. 

> Also, maybe mention the file format in this general description?

Currently, only CSV is supported but other formats may be added in the future. And given that CSV is the only supported format, `for example, CSV` will look a bit weird, too.